public/app/features/alerting/unified/hooks/useAbilities.ts

```diff
@@ -150,6 +151,30 @@ export function useAlertRuleAbilities(rule: CombinedRule, actions: AlertRuleActi
   }, [abilities, actions]);
 }
 
+export function useRulerRuleAbility(
```

Comment:
These are duplicates of `useAlertRuleAbility`, although they don't work with Combined rules but with Ruler rules and group identifiers. Eventually we should get rid of those working with Combined rules